pkg/controller/machine/controller.go

```diff
@@ -169,6 +180,37 @@ func (r *ReconcileMachine) Reconcile(ctx context.Context, request reconcile.Requ
 	// This must be a copy otherwise the referenced slice will be modified by later machine conditions changes.
 	originalConditions := conditions.DeepCopyConditions(m.Status.Conditions)
 
+	// Only enable MachineAPIMigration features if the featuregate is enabled.
+	if r.gate.Enabled(featuregate.Feature(openshiftfeatures.FeatureGateMachineAPIMigration)) {
+		// Check Status.AuthoritativeAPI
+		// If not MachineAPI. Set the paused condition true and return early.
+		if m.Status.AuthoritativeAPI != machinev1.MachineAuthorityMachineAPI {
```

Comment:
Where did we land on the handling of the empty string? I can't remember what we defined here?

Reply:
Let me dig through the resolved comments on the original PR!